.yamato/package-test.yml

```diff
@@ -0,0 +1,136 @@
+      - upm-ci project test  --project-path TestProject/Usd-Recorder --type project-tests --unity-version {{ editor.version }} --extra-utr-arg="--api-profile=NET_4_6"
+  artifacts:
+    logs:
+      paths:
+        - "upm-ci~/test-results/**/*"
+        - "upm-ci~/upm-ci.log"
+  dependencies:
+    - .yamato/package-pack.yml#pack
+{% endfor %}
+{% endfor %}
+
+{% for backend in test_backends %}
+{% for platform in test_platforms %}
+test_standalone_{{ platform.name }}_{{ backend.editor }}_{{ backend.name }}:
+  name : Test standalone {{backend.editor}} {{ backend.name }} {{ platform.name }}
+  agent:
+    type: {{ platform.type }}
+    image: {{ platform.image }}
+    flavor: {{ platform.flavor}}
+  variables:
+    UPMCI_PKG: "upm-ci-utils@stable"
+  commands:
+    - npm install {% if platform.name == "win" %}"%UPMCI_PKG%"{% else %}"$UPMCI_PKG"{% endif %} -g --registry {{ registry.npm }}
+    - upm-ci package test --unity-version {{ backend.editor }} --package-path package/com.unity.formats.usd
+      --platform standalone --backend {{ backend.name }} --extra-utr-arg="--testfilter=StandaloneSanityTest {{ backend.apiprofile }}"
+  artifacts:
+    logs:
+      paths:
+        - "upm-ci~/test-results/**/*"
+        - "upm-ci~/upm-ci.log"
+  dependencies:
+    - .yamato/package-pack.yml#pack
+{% endfor %}
+{% endfor %}
+
+{% for editor in promotion_test_editors -%}
+{% for platform in test_platforms -%}
+validate_{{ platform.name }}_{{ editor.version }}:
+  name : Validate {{ editor.version }} on {{ platform.name }}
+  agent:
+    type: {{ platform.type }}
+    image: {{ platform.image }}
+    flavor: {{ platform.flavor}}
+  variables:
+    UPMCI_PKG: "upm-ci-utils@stable"
+  commands:
+    - npm install {% if platform.name == "win" %}"%UPMCI_PKG%"{% else %}"$UPMCI_PKG"{% endif %} -g --registry {{ registry.npm }}
+    - upm-ci package test -u {{ editor.version }} --package-path package/com.unity.usd.core --type vetting-tests --platform editmode --extra-create-project-arg=-upmNoDefaultPackages
+  artifacts:
+    logs:
+      paths:
+        - "upm-ci~/test-results/**/*"
+        - "upm-ci~/upm-ci.log"
+  dependencies:
+    - .yamato/package-pack.yml#pack
+{% endfor -%}
+{% endfor -%}
+
+test_trigger:
+  name: Package Tests Trigger
+  triggers:
+    branches:
+      only:
+        - "/.*/"
+      except:
+        - stable
+  dependencies:
+    - .yamato/package-pack.yml#pack
+{% for editor in test_editors %}
+{% for platform in test_platforms %}
+    - .yamato/package-test.yml#test_{{platform.name}}_{{editor.version}}
+    - .yamato/package-test.yml#test_recorder_{{platform.name}}_{{editor.version}}
+{% endfor %}
+{% endfor %}
+{% for backend in test_backends %}
+{% for platform in test_platforms %}
+    - .yamato/package-test.yml#test_standalone_{{ platform.name }}_{{ backend.editor }}_{{ backend.name }}
+{% endfor %}
+{% endfor %}
+    - .yamato/formatting.yml#formatting
```

Comment:
Perhaps this would be good to rename package-formatting.yml for consistency? There is a package-coverage.yml in the templates, we could try and contribute this to the templates

Reply:
Contributing this back is a great idea. I'll find out who I need to speak to to suggest it :)